SparkPost/2web2ui
# FE-1219 - Alerts table mute button state gets reverted on first click

[FE-1219](https://sparkpost.atlassian.net/browse/FE-1219)

### What Changed
- Refactors Alerts list page to use functional components, hooks
- Fixes styling problems with "Recent Incidents" Panels - mobile <-- This was actually a little bugged - not functionally, but from a presentation standpoint

### How To Test
- Go to `/alerts` and verify the page renders and works as expected in both Hibana and the OG theme
- Create a new alert and toggle the "Mute" switch - verify it handles state correctly <-- this part is the bug fix.

### To Do
